api/v1alpha1/deviceservice_types.go

```diff
@@ -18,6 +18,15 @@ package v1alpha1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha4"
```

Comment:
need to update copyright: kubernetes --> OpenYurt

Reply:
Thanks for reminding, the license header is changed.